include/picongpu/fields/numericalCellTypes/EMFCenteredCell.hpp

```diff
@@ -112,6 +119,10 @@ namespace traits
             typedef VectorVector3D3V type;
         };
 
+        HDINLINE FieldPosition()
```

Comment:
In c++11 it should be `FieldPosition = default;` Please to not add a prefix qualifier else we will get nvcc or clang warnings.

Reply:
Ohh missread your comment. `= default` is equal to the old c++98 way.
Whe can check it in a mini ap with std::is_pod.

Reply:
offline discussed: `= default` is not equal to a hand written  empty constructor :-(